tas-installer/internal/kubernetes/job.go

```diff
@@ -0,0 +1,47 @@
+package kubernetes
+
+import (
+	"context"
+	"time"
+
+	v1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (kc *KubernetesClient) GetJob(namespace, jobName string) (*v1.Job, error) {
```

Comment:
I think this might only be used by the `checkSegmentBackupJob()` function in `cmd/install.go`. If that's the case, couldn't these two functions be combined into a single `DeleteJobIfExists()` function?